src/main.ts

```diff
@@ -51,6 +54,18 @@ export default function transform({
       store.push({ collection: "logs", data: { log: ethLog } });
     });
 
+    const ethReceipt = toEthReceipt({
+      transaction: ethTx,
+      logs: ethLogs,
+      event,
+      cumulativeGasUsed,
+    });
+    if (ethReceipt === null) {
```

Comment:
under which circumstances is this null?

i think we can also add it on ethTx / logs -> when can they be null?